src/chinks/VerticalAddition/VerticalAddition.java

```diff
@@ -0,0 +1,74 @@
+
+package chinks.VerticalAddition;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class VerticalAddition {
+   public class Node {
```

Comment:
You have duplicate code of Node Class in this and the above class. Can you move it to some other package say called Model and store this Node Class as BinaryTreeNode and reuse from there.
Also, a util package, under which you can have a class called BSTUtils and have the common logic of insert, delete, BFS and DFS methods there and reuse them in these classes